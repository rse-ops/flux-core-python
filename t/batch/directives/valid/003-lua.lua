#!/usr/bin/lua
-- Directives embedded in Lua script:
--
-- flux: -N1 --exclusive
--
-- flux: --output=job.out # Set output file
--
local app = require 'app'
app.run()
