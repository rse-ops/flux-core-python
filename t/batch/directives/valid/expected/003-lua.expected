SETARGS(['-N1', '--exclusive'])
SETARGS(['--output=job.out'])
